packages/components/src/components/Trigger/Trigger.test.js

```diff
@@ -16,131 +16,234 @@ import Trigger from './Trigger';
+          secretKey: 'gitlab-secret-key',
+          namespace: 'gitlab-secret-namespace'
+        },
+        eventTypes: ['gitlab-event-0', 'gitlab-event-1', 'gitlab-event-2']
+      }
     },
     {
-      name: 'webhooks-tekton-docker-registry',
-      value: 'myregistry'
+      cel: {
+        filter: 'cel-filter'
+      }
     }
-  ],
-  template: {
-    apiversion: 'v1alpha1',
-    name: 'simple-pipeline-template'
-  }
+  ]
 };
 
 describe('Trigger', () => {
-  it('should render all details', () => {
+  it('renders all details', () => {
     const props = {
       eventListenerNamespace: 'tekton-pipelines',
       trigger: fakeTrigger
     };
     const { queryByText } = renderWithRouter(<Trigger {...props} />);
     expect(queryByText(/Name/i)).toBeTruthy();
+    expect(queryByText(/my-fake-trigger/i)).toBeTruthy();
     expect(queryByText(/TriggerBinding/i)).toBeTruthy();
+    expect(queryByText(/triggerbinding-0/i)).toBeTruthy();
+    expect(queryByText(/triggerbinding-1/i)).toBeTruthy();
+    expect(queryByText(/triggerbinding-2/i)).toBeTruthy();
     expect(queryByText(/TriggerTemplate/i)).toBeTruthy();
-    expect(queryByText(/Interceptor/i)).toBeTruthy();
-    expect(queryByText(/Headers/i)).toBeTruthy();
-    expect(queryByText(/Parameters/i)).toBeTruthy();
-    expect(queryByText(/webhooks-tekton-release-name/i)).toBeTruthy();
-    expect(queryByText(/webhooks-tekton-target-namespace/i)).toBeTruthy();
-    expect(queryByText(/webhooks-tekton-service-account/i)).toBeTruthy();
-    expect(queryByText(/webhooks-tekton-git-server/i)).toBeTruthy();
-    expect(queryByText(/webhooks-tekton-git-org/i)).toBeTruthy();
-    expect(queryByText(/webhooks-tekton-git-repo/i)).toBeTruthy();
-    expect(queryByText(/mytrigger-tekton-pipelines-push-event/i)).toBeTruthy();
-    expect(queryByText(/myreleasename/i)).toBeTruthy();
-    expect(queryByText(/tekton-pipelines/i)).toBeTruthy();
-    expect(queryByText(/myregistry/i)).toBeTruthy();
-    expect(queryByText(/myrepo/i)).toBeTruthy();
-    expect(queryByText(/myorg/i)).toBeTruthy();
-    expect(queryByText(/github.com/i)).toBeTruthy();
     expect(queryByText(/simple-pipeline-template/i)).toBeTruthy();
-    expect(queryByText(/Wext-Trigger-Name/i)).toBeTruthy();
+    expect(queryByText(/Interceptors/i)).toBeTruthy();
+    // Check Webhook Interceptor
+    expect(queryByText(/(webhook)/i)).toBeTruthy();
+    expect(queryByText(/webhook-service-name/i)).toBeTruthy();
+    expect(queryByText(/Header/i)).toBeTruthy();
+    expect(queryByText(/webhook-service-namespace/i)).toBeTruthy();
     expect(queryByText(/Wext-Repository-Url/i)).toBeTruthy();
+    expect(queryByText(/https:\/\/github.com\/myorg\/myrepo/i)).toBeTruthy();
     expect(queryByText(/Wext-Incoming-Event/i)).toBeTruthy();
+    expect(queryByText(/wext-incoming-event-value/i)).toBeTruthy();
     expect(queryByText(/Wext-Secret-Name/i)).toBeTruthy();
+    expect(queryByText(/wext-secret-name-value/i)).toBeTruthy();
```

Comment:
do we need to explicitly check for all of these or can we simplify the test?

Reply:
The previous test explicitly checked for all of them... but I could simplify it by looping over the Header names & values?